e2e/test/scenarios/dashboard/reproductions/29304-scalar-first-render-size-change.cy.spec.js

```diff
@@ -0,0 +1,115 @@
+import { restore } from "e2e/support/helpers";
+
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { WAIT_TIME } from "metabase/components/ExplicitSize";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATABASE;
+
+const SCALAR_QUESTION = {
+  name: "Scalar question",
+  query: {
+    "source-table": ORDERS_ID,
+    aggregation: [["count"]],
+  },
+  display: "scalar",
+};
+
+const SCALAR_QUESTION_CARD = { size_x: 4, size_y: 3, row: 0, col: 0 };
+
+const SMART_SCALAR_QUESTION = {
+  name: "Smart scalar question",
+  query: {
+    "source-table": ORDERS_ID,
+    aggregation: [["count"]],
+    breakout: [
+      [
+        "field",
+        ORDERS.CREATED_AT,
+        {
+          "base-type": "type/DateTime",
+          "temporal-unit": "month",
+        },
+      ],
+    ],
+  },
+  display: "smartscalar",
+};
+
+const SMART_SCALAR_QUESTION_CARD = SCALAR_QUESTION_CARD;
+
+describe("issue 29304", () => {
+  describe("display: scalar", () => {
+    beforeEach(() => {
+      restore();
+      cy.signInAsAdmin();
+      cy.intercept("api/dashboard/*/dashcard/*/card/*/query").as(
+        "getDashcardQuery",
+      );
+      cy.intercept("api/dashboard/*").as("getDashboard");
+      cy.clock();
+    });
+
+    it("should render scalar with correct size on the first render (metabase#29304)", () => {
+      cy.createDashboard().then(({ body: dashboard }) => {
+        cy.createQuestionAndAddToDashboard(
+          SCALAR_QUESTION,
+          dashboard.id,
+          SCALAR_QUESTION_CARD,
+        );
+
+        visitFullAppEmbeddingUrl({ url: `/dashboard/${dashboard.id}` });
+
+        cy.wait("@getDashboard");
+        cy.wait("@getDashcardQuery");
+        cy.tick(WAIT_TIME);
+
+        const expectedWidth = 147;
+        cy.findByTestId("scalar-value").then(([$scalarValue]) => {
+          expect($scalarValue.offsetWidth).to.be.closeTo(
+            expectedWidth,
+            expectedWidth * 0.1,
+          );
```

Comment:
The previous version
https://github.com/metabase/metabase/blob/fab35e5b5b01cf3410672e4adf829fe7dfed2de3/e2e/test/scenarios/dashboard/reproductions/29304-scalar-first-render-size-change.cy.spec.js#L67
flakes on CI. Here's the [example failed run](https://www.deploysentinel.com/ci/runs/64bfb4a3857dcf477f8d637c).

So I increase the tolerance to 10%